layers/+lang/elixir/packages.el

```diff
@@ -105,7 +110,28 @@
       "cf" 'alchemist-compile-file
       "c:" 'alchemist-compile
 
-      "," 'alchemist-goto-jump-back)
+      "gg" 'alchemist-goto-definition-at-point
+      "." 'alchemist-goto-definition-at-point
+      "gb" 'alchemist-goto-jump-back
+      ","  'alchemist-goto-jump-back
+      "gN" 'alchemist-goto-jump-to-previous-def-symbol
+      "gn" 'alchemist-goto-jump-to-next-def-symbol
+      "gj" 'alchemist-goto-list-symbol-definitions
+
+      "Xi" 'alchemist-hex-info-at-point
+      "Xr" 'alchemist-hex-releases-at-point
+      "XR" 'alchemist-hex-releases
+      "XI" 'alchemist-hex-info
+      "Xs" 'alchemist-hex-search
+
+      "ol" 'alchemist-macroexpand-once-current-line
+      "oL" 'alchemist-macroexpand-once-print-current-line
+      "ok" 'alchemist-macroexpand-current-line
+      "oK" 'alchemist-macroexpand-print-current-line
+      "oi" 'alchemist-macroexpand-once-region
+      "oI" 'alchemist-macroexpand-once-print-region
+      "or" 'alchemist-macroexpand-region
+      "oR" 'alchemist-macroexpand-print-region)
```

Comment:
I don't how macroexpand works for elixir but do you think this could be similar to `macrostep` for elisp ?
https://github.com/syl20bnr/spacemacs/blob/develop/layers/+lang/emacs-lisp/packages.el#L164-L175